PassengersRepository.cs

```diff
@@ -0,0 +1,64 @@
+﻿using AirportTicketBooking.Models;
+
+namespace AirportTicketBooking.Repository
+{
+    public static class PassengersRepository
```

Comment:
Move repos to a folder called `Repositories`

Reply:
done